src/main/java/eggventory/commons/enums/StockProperty.java

```diff
@@ -1,5 +1,6 @@
-package eggventory.enums;
+package eggventory.commons.enums;
 
+//@@author patwaririshab
 public enum StockProperty {
```

Comment:
still got to resolve this StockProperty and Property are the same from when Raghav changed it in #249